store.css

```diff
@@ -0,0 +1,243 @@
+/* --==================== general styles ====================-- */
+body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+
+h1, h2, .about-text{
+  font-weight: lighter;
+}
+
+a{
+  text-decoration: none;
+}
+
+ /* --==================== HEADER ====================-- */
+ .header_logo {
+    height: 40px;
+    float: left;
+    position: relative;
+    padding-left: 100px;
+    /*margin: 10px 15px 15px 10px;*/
+  }
+
+  #header-store{
+    background: rgba(196, 196, 196, 0.1);
+  }
+
+ /* --==================== NAVBAR ====================-- */
+ 
+ .nav_icon-text{
+  font-weight: 450;
+  color:#6b6b6b;
+  padding: 15px;
+}
+ 
+ .nav_icon{
+  font-weight: 450;
+  color:#9c9a9a;
+  padding: 15px;
+ }
+
+ .nav_icon:hover{
+   color: #d65924;
+
+ }
+
+ .container{
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin-right: 60px;
+  list-style: none;
+ }
+
+ /* --==================== FEATURE SECTION ====================-- */
+
+*{
+    margin: 0;
+    padding: 0;
+}
+/*
+body{
+    background: #c4e0e5;
+}*/
+.intro{
+    font-weight: 500;
+    font-style: normal;
+    font-size: 50px;
+    color: #DE6E49;
+  }
+
+  .parent{
+    width: 100%;
+    display: flex;
+    /*justify-content: center;*/
+  }
+  
+  .background-image{
+    width: 50%;
+    height: auto;
+  }
+
+  .store-image{
+    width: 600px;
+    height: auto;
+    float: right;
+  }
+  
+  
+  
+
+
+
+/* --==================== FORM SECTION ====================-- */
+
+.wrapper{
+    width: 50%;
+    /*max-width: 680px;*/
+    /*margin: 100px auto 0;*/
+    padding-left: 100px;
+    padding-top: 100px;
+    box-sizing: border-box;
+}
+
+.content h1{
+    letter-spacing: 1px;
+}
+
+.form{
+    width: 100%;
+    margin-top: 25px 0;
+}
+
+.top-form, 
+.middle-form,
+.bottom-form{
+    width: 100%;
+    min-height: 65px;
+    margin: 25px 0;
+}
+
+.form input[type="text"],
+[type="select"]{
+    padding: 10px 10px; 
+    box-sizing: border-box;
+    width: 100%;
+    border: 2px solid #545858; /* the width or length of the text area*/
+    border-radius: 4px;
+    outline: none;
+    transition: all 0.2s ease;
+    font-size: 12px;
+}
+
+.form input:focus,
+.form select:focus{
+    border-color: #ffda0a;
+    box-shadow: inset 0 1px 1px rgba(0,0,0, 0.0125),
+                0 0 8px rgba(76,161,175,0.5);
+}
+
+.form .label{
+    margin-bottom: 5px;
+}
+
+.label{
+    color: #545858;
+}
+
+.top-form .inner-form{
+    width: 47.5%; /* width of both names*/
+    float: left;
```

Comment:
Please don't use `float` rules in your CSS. Float is the old way of making layouts in CSS and it has now been replaced by Flexbox (which was introduced a few years ago). 
Set `display: flex` on the parent of the element, in this case `.top-form`